pkg/controller/efs/filesystem/setup.go

```diff
@@ -96,12 +146,41 @@ func postObserve(_ context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.Desc
 	return obs, nil
 }
 
-func preUpdate(_ context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.UpdateFileSystemInput) error {
+func (e *custom) preUpdate(ctx context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.UpdateFileSystemInput) error {
 	obj.FileSystemId = pointer.ToOrNilIfZeroValue(meta.GetExternalName(cr))
 	// Type of this field is *float64 but in practice, only integer values are allowed.
 	if cr.Spec.ForProvider.ProvisionedThroughputInMibps != nil {
 		obj.ProvisionedThroughputInMibps = aws.Float64(float64(pointer.Int64Value(cr.Spec.ForProvider.ProvisionedThroughputInMibps)))
 	}
+
+	if e.cache.backupPolicy != *cr.Spec.ForProvider.Backup {
+		var policy *string
+		if pointer.BoolValue(cr.Spec.ForProvider.Backup) {
+			policy = aws.String("ENABLED")
+		} else {
+			policy = aws.String("DISABLED")
+		}
+
+		if pointer.StringValue(obj.FileSystemId) == "" {
+			obj.FileSystemId = pointer.ToOrNilIfZeroValue(meta.GetExternalName(cr))
+		}
+		_, err := e.client.PutBackupPolicyWithContext(ctx,
+			&svcsdk.PutBackupPolicyInput{
+				FileSystemId: obj.FileSystemId,
+				BackupPolicy: &svcsdk.BackupPolicy{Status: policy}})
```

Comment:
For consistency the code formatting should be same as everywhere else: Newline before `}`.